### ISSUE TITLE
Upload of files with spaces in the name

### ISSUE THREAD

**Issue by ccavanaugh** [minad/3delta#1]
![snapshot2](https://cloud.githubusercontent.com/assets/636962/7575297/b438f736-f801-11e4-9264-3f0359bc4d85.png)
Files appear to upload fine if they have spaces in the file name, but they do not display correctly in the Player box.  The right most part of the file name after any spaces is displayed instead of the complete file name.

In the command shell, Smoothie reports the correct file name

**Comment by minad** [comment]
Yes, this is a bug. I have to improve the parsing of the filenames from the M20 command.
